pep-9999.txt

```diff
@@ -0,0 +1,171 @@
+PEP: 9999
+Title: Python local packages directory [We need to find a proper name]
+Version: $Revision$
+Last-Modified: $Date$
+Author: Kushal Das <mail@kushaldas.in>, Steve Dower <steve.dower@python.org>,
+        Donald Stufft <donald@stufft.io>, Nick Coghlan <ncoghlan@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 16-May-2018
+Python-Version: 3.8
+
+
+
+Abstract
+========
+
+This PEP proposes to add to Python a mechanism to have a ``__pypackages__``
+directory  per project to have its own independent set of installed packages.
```

Comment:
How about "... to automatically recognize a `__pypackages__` directory and prefer importing packages installed in this location over user or global site-packages". For me, this:
* avoids mentioning "project" (which is not well defined)
* defines "independent" (implicitly, and we'll probably have to mention it earlier)
* leaves us space to later handle a `pyvenv.cfg` differently, since all we've said we'll do is "recognize" it